src/app/state/staking/saga.ts

```diff
@@ -84,7 +84,14 @@ function* loadDebondingDelegations(publicKey: Uint8Array) {
 
 function* refreshValidators() {
   const { accounts } = yield* call(getExplorerAPIs)
-  const validators = yield* call([accounts, accounts.getValidatorsList], {})
+  let validators
+  try {
+    validators = yield* call([accounts, accounts.getValidatorsList], {})
+  } catch (e) {
+    console.error('get validators list failed, continuing without updated list.', e)
+    yield* put(stakingActions.updateValidatorsError('' + e))
```

Comment:
could perhaps use `e.stack` here if:

1. we would know that e instanceof Error
2. we render in a `<pre>`